examples/attention_sink/example_mha_sink_bwd_bhsd.py

```
@@ -492,6 +492,46 @@ def tl_bwd():
+    window_size: Optional[int] = None,
+    dtype: str = "float16",
+):
+    torch_dtype = {"float16": torch.float16, "bfloat16": torch.bfloat16}[dtype]
+    with torch.no_grad():
+        Q = torch.randn(BATCH, H, N_CTX, D_HEAD, dtype=torch_dtype, device="cuda")
+        K = torch.randn_like(Q)
+        V = torch.randn_like(Q)
+        sinks = torch.randn(H, dtype=torch_dtype, device=Q.device)
+        dO = torch.randn_like(Q)
+        fwd = flashattn_fwd(BATCH, H, N_CTX, D_HEAD, window_size=window_size, dtype=dtype)
+        O, lse = fwd(Q, K, V, sinks)
```

Ruff (0.14.8)

Ambiguous variable name: `O`

(E741)